content/docs/instrumenting/writing_exporters.md

```diff
@@ -461,6 +461,9 @@ putting your username and WIP is fine.
 This is a registry to make our users’ lives a little easier, not a commitment
 to develop particular exporters.
 
+For exporters on internal applications we recommend using ports outside of the
```

Comment:
Users in such a situation will likely be following their internal port number policy, I think this text would only confuse them into thinking they needed to follow the Prometheus suggested ports rather than their own org's policy.